packages/asc-ui/src/components/Alert/Alert.stories.tsx

```diff
@@ -48,7 +52,7 @@ export const CompactState = () => (
       dismissible
       level="attention"
     />
-    <Alert heading="Only heading with button" compact level="error" />
```

Comment:
The error alert has been removed here, is that intentional?

Reply:
I just changed `error` to `warning`